apps/ui/src/views/Space/Overview.vue

```diff
@@ -116,6 +142,28 @@ watchEffect(() => setTitle(props.space.name));
         </div>
       </div>
     </div>
+    <template v-if="space.children.length">
+      <UiLabel :label="'Sub-spaces'" />
+      <div class="relative">
+        <div
+          class="bg-gradient-to-r from-skin-bg left-0 top-0 bottom-0 w-3 absolute z-10 pointer-events-none"
+        />
+        <div class="overflow-x-scroll no-scrollbar">
+          <div class="px-4 py-3 flex gap-3" data-no-sidebar-swipe>
+            <SpacesListItem
+              v-for="child in space.children"
+              :key="child.id"
+              :space="child"
+              :show-about="false"
+              class="basis-[230px] shrink-0"
+            />
+          </div>
```

Comment:
In the start we have this gap 
<img width="470" alt="Untitled 2" src="https://github.com/user-attachments/assets/24e26505-5eb9-4d69-8a1a-b46ac81b48ce">

But in the end, it is missing the gap and covered with the blur filter 
<img width="375" alt="Untitled 5" src="https://github.com/user-attachments/assets/ae3b1df3-996e-4254-95cd-442c84ed5f31">

I think we should have the same gap here also (would suggest it is end of the list)

Reply:
mhhhh ... the `px-4` is not working, weird